src/lints/enum_repr_int_changed.ron

```diff
@@ -43,9 +49,14 @@ SemverQuery(
                         # - but is not the same repr(i*) or repr(u*) that we had before.
                         # This is the breaking change.
                         attribute @fold @transform(op: "count") @filter(op: ">", value: ["$zero"]) {
-                            value @filter(op: "regex", value: ["$repr_regex"])
-                                    @filter(op: "!=", value: ["%attr_value"])
-                                    @output(name: "new_attr")
+                            as_string @output(name: "new_attr")
+                            content {
+                                base @filter(op: "=", value: ["$repr"])
+                                argument {
+                                    base @filter(op: "regex", value: ["$repr_regex"])
+                                            @filter(op: "!=", value: ["%old_attr_value"])
```

Comment:
```suggestion
                                    base @filter(op: "regex", value: ["$repr_regex"])
                                         @filter(op: "!=", value: ["%old_attr_value"])
```
Another minor alignment fix. Ideally one day we'll have an auto-formatter for this.